### ISSUE TITLE
CPDBear is too noisy by default for PHP

### ISSUE THREAD

**Issue by jayvdb** [coala-analyzer/coala-bears#810]
When using CPDBear (from the docker image) on two PHP projects I have found many of the CPDBear results to be silly, and thus my first instinct was to disable it.

For example, this is the default output.
https://travis-ci.org/jayvdb/clipdig/builds/159077259
```
form/module/beritasatu/get_content.php

|   1| <?php

form/pending/gatra/get_content.php

|   1| <?php

|    | [NORMAL] CPDBear:

|    | Duplicate code found.

form/module/detik/get_content.php

|   1| <?php

form/module/sinarharapan/get_content.php

|   1| <?php

|    | [NORMAL] CPDBear:

|    | Duplicate code found.
```

Setting `minimum_tokens = 500` in the `.coafile` doesnt make the problem disappear.

This might be a problem upstream in pmd.
It might be possible to make it less silly with other configuration options.
If not, it might be best to disable it for PHP until it shows meaningful results.

**Comment by sils1297** [comment]
Hi, the problem here is actually that we don't show the context of the lines due to a bug, the duplication is there and if you look into the files you'll likely see the duplication.

The bug has been fixed, I'll make a bugfix release soon including this. In the meantime you can install the bears package with `--pre` to get the prerelease.

**Comment by jayvdb** [comment]
There certainly is a lot of duplication in this codebase, so it would be lovely if it was reported sanely.

I'll try upgrading the bears in the docker; that will be useful for the documentation.

**Comment by sils1297** [comment]
I'll try to get it into coala/base:0.8 today

**Comment by jayvdb** [comment]
Need to remove the "upstream" label on this ticket?

**Comment by sils1297** [comment]
Not sure yet. If the "shwing the right source ranges in coala" solves the problem for you then this has been fixed on master and this can be closed.

**Comment by sils1297** [comment]
Reopen if needed

**Comment by jayvdb** [comment]
Confirmed fixed in master.

**Comment by sils1297** [comment]
:+1: I just did a release with the fix so you'll be able to use coala 0.8.4. I just triggered a rebuild of the docker image.

**Comment by sils1297** [comment]
@jayvdb btw. we aren't fully sure about the default for the minimum tokens setting, if you have suggestions on how to change it please let us know!
